[TM] Chp2-2  Update and center the chapter on first map loading and creation (#3298)
qgis/QGIS-Documentation: source/docs/training_manual/introduction/preparation.rst

Let's get started right away!

|basic| |FA| Prepare a map
-------------------------------------------------------------------------------

#. Open QGIS. You will have a new, blank map.

> #. Look for and click the :guilabel:`Open Data Source Manager` button: |dataSourceManager|.

Reviewer: ```suggestion
#. Click the :guilabel:`Open Data Source Manager` button: |dataSourceManager|.
```

   If you can't find the icon, check that the :guilabel:`Data Source Manager`
   toolbar is enabled in the :menuselection:`View --> Toolbars` menu.

   The :guilabel:`Data Source Manager` dialog allows you to choose the data to
   load depending on the data type:

>    the tabs in the left part of the dialog list the main data types.

Reviewer: ```suggestion
   the tabs in the left-hand section of the dialog list the main data types.
```

>    each tab on the left part of the dialog lists the main data types.

Reviewer 2: ```suggestion
   the tabs in the left part of the dialog list the main data types.
```

   You can also browse the files directly from the folder of your computer
   with the *Browser* tab (see :ref:`browser_panel_tm` section):

   .. image:: img/add_data_dialog.png
      :align: center
      :scale: 60 %

#. Click on the :guilabel:`Vector` tab, enable the |radioButtonOn|:guilabel:`File`
   source type and press the :guilabel:`...` button next to :guilabel:`Vector
   Dataset(s)`.
#. Navigate to the file :file:`exercise_data/shapefile/protected_areas.shp`
   (in your course directory).
#. Select the file and click :guilabel:`Open`. You will see the original dialog,
   but with the file path filled in.

   .. image:: img/add_vector_dialog.png
      :align: center

#. Click :guilabel:`Add` here as well. The data you specified will now load.

Congratulations! You now have a basic map. Now would be a good time to save
your work.

#. Click on the :guilabel:`Save As` button: |fileSaveAs|
#. Save the map under a :file:`solution` folder next to :file:`exercise_data`
   and call it :file:`basic_map.qgs`.

.. _backlink-interface-preparation-1:

> |basic| |TY|

Reviewer 2: Travis complains: `Undefined substitution referenced: "TY".`

-------------------------------------------------------------------------------

> Repeating the steps above add the :file:`places.shp` and :file:`rivers.shp`

Reviewer 2: ```suggestion
Repeat the steps above to add the :file:`places.shp` and :file:`rivers.shp`
```

> files from the same :file:`exercise_data/shapefile` folder to the map.

Reviewer 2: ```suggestion
data sets from the same folder (:file:`exercise_data/shapefile`) to the map.
```

> data sets from the same folder (:file:`exercise_data/shapefile`) to the map.

Reviewer: datasets/data sets?

|WN|
-------------------------------------------------------------------------------

Now you're familiar with the basic function of the :guilabel:`Open Data Source Manager`
button, but what about all the others? How does this interface work? Before we

> go on with the more involved stuff, let's first take a good look at the general

Reviewer: ```suggestion
go on, let's first take a look at the
```
